.env

```diff
@@ -3,4 +3,6 @@ MAILER_PASSWORD=f1817eab4f1cc5
 MAILER_USER_NAME=ccd3b4d9d5e93a
 MAILER_ADDRESS=smtp.mailtrap.io
 MAILER_PORT=2525
-MAILER_AUTHENTICATION=cram_md5
\ No newline at end of file
+MAILER_AUTHENTICATION=cram_md5
```

Comment:
I've just noted this.... you are pushing the .env file! we need to remove this from the history asap

Reply:
WHAT